contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,244 @@
+# Proposal to Standardize Development Process
+
+## Links
+```
+---
+Links:
+- Issues:
+  - [someIssueURL]()
+- PRs:
+  - https://github.com/kubernetes/community/pull/967
+- MailingListDiscussions:
+  - https://groups.google.com/forum/#!topic/kubernetes-dev/65A-3ULYPB0
+- Documentation:
+  - [someDocsLinkURL]()
+```
+
+## Responsible SIG(s)
+
+- SIG Release
+- SIG PM
+- SIG Architecture
+- SIG Governance
+- Technical Steering Committee
+
+## Summary
+
+A standardized development process for Kubernetes is proposed in order to:
+- provide a common structure for proposing changes to Kubernetes
+- ensure that the motivation for a change is clear
+- allow for the enumeration stability milestones and stability graduation
+  criteria
+- persist project information in a Version Control System (VCS) for future
+  Kubernauts
+- support the creation of _high value user facing_ information such as:
+  - release notes
+  - release announcement blog
+  - a cross project development road map
```

Comment:
There are going to be things that we want to do (on a road map) that aren't yet codified or described in a proposal.  Often time the line items on a road map are too ill defined for even a stub proposal (we see this with some of the oldest bugs).  I'd rather we have a separate process (which points to proposals) for the road map.